### PR TITLE
Assignment 1 - Roberto Garcia

### PR DESCRIPTION
**Pull request by robertogarcia11**
fixed two bugs,
1] stored question into array
2] changed int to float in percentage calculation

### CODE REVIEW

**Comment by timmytaters** [comment]
Starting review

**Review by timmytaters — COMMENTED** [review]
Looks good, not much change/new things but runs/builds fine and passes all tests

Assignment 1 Review of Roberto G by Timothy N
Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit N/A
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌
Issue 3: Feature request unit tested ❌

**Comment by omar5o9** [comment]
Code looks good I did something similar to the way you fixed the percentage calculation.

Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit: No partner
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌
Issue 3: Feature request unit tested ❌

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static void Main(string[] args)
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
     }
 
-    public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+    public static string GetPercentCorrect(float numberCorrectAnswers, int numberOfQuestions)
```
Comment (omar5o9) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:22]: I did the same thing here, but you could've casted the whole thing as a float on the return line

**Pull request closed without merging**